recipes/tvips-tools/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "TVIPS tools" %}
+{% set tarname = name|lower|replace(' ', '-') %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ tarname }}
+  version: {{ version }}
+
+source:
+  url: https://cryoem.ucla.edu/downloads/{{ tarname }}-{{ version }}.tar.gz
+  sha256: 87b5416470e4fa93b08c97ae8c73f71552945058e7f3a963e64248530646403d
+  patches:
+    - 01-setenv.patch
+    - 02-version.patch
+    - 03-warnings.patch
+
+build:
```

```suggestion
build:
   ignore_run_exports:
   - libstdcxx-ng
   - libcxx
```

---

Either

```suggestion
build:
  ignore_run_exports_from:
   - {{ compiler('cxx') }}
```

or do you really need the C++ compiler here? Could you simply omit it from `run`

---

CXX is needed, but not the C++ standard library (that is pretty rare, I suspect)